### PR TITLE
Fix cbvalv (fixes #2388)

### LINKED ISSUE #2388: cbvalv and ax-10
The comment above `cbvalv` says, in part, "Remove dependency on ~ ax-10 ."  However, that is no longer the case:
```
MM-PA> show trace_back cbvalv /essential /match ax*
The proof of statement "cbvalv" uses the following earlier statements:
  ax-mp($a) ax-1($a) ax-2($a) ax-3($a) ax-gen($a) ax-4($a) ax-5($a) ax5e ax5ea
  ax-6($a) ax6v ax6ev ax-7($a) ax7v ax7v1 ax7v2 ax6evr ax7 ax-10($a) ax-11($a)
  ax-12($a) ax12v ax-13($a) ax13v ax13lem1 ax6e
```

A git bisect shows that the dependency on ax-10 appeared with commit https://github.com/metamath/set.mm/commit/6f7ee0b388d3654dad7c3878522b0866808c43ca.  Either the comment is wrong and should be removed (and the proof can be shortened to simply invoke `cbval`), or the dependency on `ax-10` should be removed again.  The dependency chain is `cbvalv` -> `nfal` -> `nf5i` -> `nf5-1`-> `hbe1a` -> `hbn1` -> `ax-10`.

### PR DESCRIPTION
**Pull request by wlammen**
Thanks to @jamesjer who noticed this oversight.
@tirix Rewrap issues a warning that seems to be related to [Nathanson] p. 123
?Warning: Bibliography keyword missing in comment for "breprexp".
    (See HELP WRITE BIBLIOGRAPHY for list of keywords.)

### CODE REVIEW

**Review by benjub — APPROVED** [review]

**Comment by benjub** [comment]
@wlammen : can you also do cbv3 ? The change in cbvalv will remove ax-10-dependency from cbvaldva, cbvexdva, cbval2v, cbvex2v.

**Comment by avekens** [comment]
The "Bibliography keyword missing" issue is already discussed in #2377 ...

**Review by avekens — APPROVED** [review]

**Review by benjub — APPROVED** [review]

**Comment by tirix** [comment]
> The "Bibliography keyword missing" issue is already discussed in #2377 ...

That fix is in #2379 waiting for your approval! ;)

**Pull request merged**
